packages/api/src/routers/api/alerts.ts

```diff
@@ -43,6 +46,13 @@ const zAlert = z
 
 const zAlertInput = zAlert;
 
+const getHistory = async (alertId: string, teamId: string) => {
+  const histories = await AlertHistory.find({ alert: alertId, team: teamId })
+    .sort({ createdAt: -1 })
+    .limit(20);
```

Comment:
I wonder if we want to pull history by time frame (as an option). that's something we can think about once the frontend draft is out

Reply:
Also I realize we might want to index this query